File: 01_ProgrammingBasics/_05_LoopsPartTwo/_02_Password.java
package _01_ProgrammingBasics._05_LoopsPartTwo;

import java.util.Scanner;

public class _02_Password {
    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);

        // Read input values
        String username = scan.nextLine();
        String password = scan.nextLine();

        String input = scan.nextLine();

        // Use a while loop to repeatedly ask for the password until it matches
        while (!input.equals(password)) {
            input = scan.nextLine();
        }

        // Welcome the user after successful login
        System.out.printf("Welcome %s!", username);

        scan.close();
    }
}

File: 23_C#Basics/_05_LoopsPartTwo/_02_Password/_02_Password.cs
using System;

public class _02_Password
{
    public static void Main(string[] args)
    {
        // Read input values
        string username = Console.ReadLine();
        string password = Console.ReadLine();

        string input = Console.ReadLine();

        // Use a while loop to repeatedly ask for the password until it matches
        while (input != password)
        {
            input = Console.ReadLine();
        }

        // Welcome the user after successful login
        Console.WriteLine("Welcome {0}!", username);
    }
}
